parser/process_dataset.py

```
@@ -596,22 +596,25 @@ def convert_dir(
+            if schema_validate(os.path.join(local_dir, file)):
+                id_parser = MzIdParser(
+                    os.path.join(local_dir, file),
+                    peaklist_dir,
+                    writer,
+                    logger,
+                )
+                try:
+                    id_parser.parse()
+                    # logger.info(id_parser.warnings + "\n")
+                except Exception as e:
+                    logger.error(f"Error parsing {file}")
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)